eclass/texlive-module.eclass

```diff
@@ -141,9 +138,9 @@ texlive-module_src_unpack() {
 	unpack ${A}
 
 	grep RELOC tlpkg/tlpobj/* | awk '{print $2}' | sed 's#^RELOC/##' > "${T}/reloclist"
-	{ for i in $(<"${T}/reloclist"); do  dirname $i; done; } | uniq > "${T}/dirlist"
+	{ for i in $(<"${T}/reloclist"); do  dirname ${i}; done; } | uniq > "${T}/dirlist"
```

Comment:
You don't have to use `{ ... }` here. Also, this is awful and dangerous. You should use `read` to read files.

Reply:
@mgorny `src_unpack()` is entirely PITA, it must be also speed up, I will leave it as is untill tl2020, which is in 2 months in anyway.